LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs

```diff
@@ -189,7 +191,7 @@ internal static S7Block GetAWLBlockBasicInfoFromBlockHeader(byte[] MC7Code, int
                  * xx      = Nertworks
                  */
 
-                retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);
+                //retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);

```

Comment:
I think i got confused by searching for my problem with the Version field of Online blocks. I confused Version with BlockVersion. I think this line should be reverted to be the same as before.

Anyway What is stored on those Bytes?